module.json

```
@@ -2,7 +2,7 @@
   "name": "module",
   "title": "New Module",
   "description": "",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

```suggestion
  "version": "0.0.1",
```
This should be left as one, as that will be the first version the new repo will build as